lib/plan-builder-generated.js

```diff
@@ -8540,7 +8760,7 @@ fromDocDescriptors(...args) {
 
     }
 /**
-    * This function matches and returns the uri, content, and score for documents. Provides a client interface to a server function. See {@link http://docs.marklogic.com/op.fromDocUris|op.fromDocUris}
+    * This function matches and returns the uri for documents. Provides a client interface to a server function. See {@link http://docs.marklogic.com/op.fromDocUris|op.fromDocUris}
```

Comment:
This is fine for now, but in the future, any uses of "uri" (unless referring to a column name) should be changed in the hint/codegen file to be "URI".